dbus-codegen/src/main.rs

```diff
@@ -56,6 +58,10 @@ Defaults to 'RefClosure'."))
         let c = dbus::Connection::get_private(bus).unwrap();
         let p = c.with_path(dest, path, 10000);
         p.introspect().unwrap()
+    } else if let Some(_file_path) = matches.value_of("file")  {
+    	let mut s = String::new();
+    	s=std::fs::read_to_string(matches.value_of("file").unwrap().to_string()).unwrap();
```

Comment:
* this unwrap can be avoided if you just use the value you unpack above if the if condition.
* Some space before and after the `=` would be nice.

Reply:
Actually you don't need `s` at all.